packages/rpc/src/util.ts

```diff
@@ -1,40 +1,50 @@
-import { mbjs, RPC_ENDPOINTS, NEAR_RPC_ENDPOINTS } from '@mintbase-js/sdk';
 import fetch from 'cross-fetch';
 
 
-export type RPC_OPTIONS  = 'lava' | 'near' | 'beta'
+
+export const queryNearRpc = async ({params, method, rpcUrl}):  Promise<{ result: Record<string, unknown>, error: unknown } | undefined> => {
+  return await requestFromNearRpc({
+    jsonrpc: '2.0',
+    id: 'dontcare',
+    method:method,
```

Comment:
If the name here is `queryNearRpc`, why don't we fill in `method: 'query'` and save typing on usage?

Reply:
because its just a wrapper and we use different methods on the calls